samples/notes/android/src/main/kotlin/com/dropbox/market/notes/android/data/api/FakeApi.kt

```diff
@@ -0,0 +1,48 @@
+package com.dropbox.market.notes.android.data.api
+
+import com.dropbox.market.notes.android.Key
+import com.dropbox.market.notes.android.Note
+import com.dropbox.market.notes.android.data.Notebook
+import com.dropbox.market.notes.android.data.MarketNote
+import com.dropbox.market.notes.android.data.asMarketNote
+import io.ktor.client.HttpClient
+import io.ktor.client.request.get
+import io.ktor.http.HttpStatusCode
+import kotlin.collections.set
+
+class FakeApi(private val client: HttpClient) : Api<Key, Notebook> {
```

Comment:
You can move this and the other Fake implementations out of the main source path and into the relevant test path